src/whylogs/core/image_profiling.py

```diff
@@ -58,7 +50,7 @@ class TrackImage:
 
     def __init__(self,
                  filepath: str = None,
-                 img: ImageType = None,
+                 img=None,
```

Comment:
Type checked removed why ?

Reply:
hard import on PIL :). This is preventing `import whylogs` (see the attached issue)

Reply:
Using try catch and now we can keep the type :)